### PR TITLE
fix(slider): Added touchevents and eventemitter to listen for value change

### PR DESCRIPTION
**Pull request by albinscania**
**Describe pull-request**  
_Describe what the pull-request is about_
Added missing touchevents so Slider can be used on mobile.
Added a custom change event so it's programatically possible to listen to value updates.
Updared story to reflect the new eventemitter.

Fixes: 2248

**How to test**  
Checkout the branch, check the eventlistener (it will output to console). Put chrome in mobile mode and use the slider.

### CODE REVIEW

**Comment by alexlioliopoulos** [comment]
Touch works great!

**Review by EmelieLitwin — APPROVED** [review]
Looks good!

File: components/src/components/slider/slider.tsx [hunk 1]
```diff
@@ -63,12 +71,24 @@ export class Slider {
   /** Name property (will be inherited by the native slider component) */
   @Prop() name = '';
 
-  /** Decide to use the small variant or not */
+  /** @DEPRECATED Decide to use the small variant or not */
   @Prop() small = null;
```
Comment (theJohnnyMe) [inline components/src/components/slider/slider.tsx:75]: Can you please assign types to all props, like boolean, string etc and default value in that case?
Maybe a bit out of the scope of this task, but other colleagues integrating this one will appreciate it a lot.

Comment (albinscania) [reply components/src/components/slider/slider.tsx:75]: Sure! Will do

**Review by theJohnnyMe — APPROVED** [review]

**Comment by EmelieLitwin** [comment]
Just a suggestion but maybe hidet he tick-related controls in SB (number of ticks, tick numbers and snap to ticks) if show ticks is false? There are so many options and it wouldn't hurt to hide some if they can't be used.

**Pull request merged**